src/index.html

```diff
@@ -1,13 +1,392 @@
-    <title>Miami</title>
+    <title>NOTHING</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@400;700&family=Space+Mono&display=swap"
+      rel="stylesheet">
+    <link
+      rel="shortcut icon"
+      href="./images/icons/favicon_32x32.webp"
+      type="image/x-icon"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+
+  <body class="page__body">
+    <header class="page__header header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a class="top-bar__logo-link" href="#">
+              <img
+                class="top-bar__logo"
+                src="./images/Logo.png"
+                alt="NOTHING logo"
+              >
+            </a>
+            <div class="top-bar__icons icons">
+              <a href="tel:+1 234 5555-55-55" class="icon icon--phone"></a>
+              <a href="#menu" class="icon icon--menu"></a>
+            </div>
+          </div>
+          <h1 class="header__title">
+            Bring joy back<br>to the everyday
+          </h1>
+        </div>
+      </div>
+    </header>
+
+    <aside class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="top-bar menu__top-bar">
+          <a class="top-bar__logo-link" href="#">
+            <img
+              class="top-bar__logo"
+              src="./images/Logo.png"
+              alt="NOTHING logo"
+            >
+          </a>
+
+          <div class="top-bar__icons">
+            <a href="#" class="icon icon--close"></a>
+          </div>
+        </div>
+
+        <div class="menu__wrapper">
+          <nav class="menu__nav nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a class="nav__link" href="#recommended">Recommended</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#categories">Store</a>
+              </li>
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="./blank-page.html"
+                  target="_blank"
+                >
+                  Login
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="./blank-page.html"
+                  target="_blank"
+                >
+                  Cart (0)
+                </a>
+              </li>
+            </ul>
+          </nav>
+          <div class="menu__phone-wrapper">
+            <a
+              class="menu__phone-number"
+              href="tel:+1 234 5555-55-55"
```

Comment:
Need to fix format phone everywhere only in href
```suggestion
              href="tel:+123455555555"
```